index.js

```diff
@@ -116,9 +116,13 @@ class Installer {
         })
       }).tap(full => {
         this.pkgCount++
```

Comment:
Would be nice if you added a `this.failedPkgCount`, and incremented that, maybe? Would be good to report it...